NAMESPACE

```diff
@@ -34,11 +35,15 @@ export(renderTranslateExecuteSql)
 export(renderTranslateQueryApplyBatched)
 export(renderTranslateQuerySql)
 export(renderTranslateQuerySqlToAndromeda)
+exportClasses("Microsoft SQL Server")
```

Comment:
In general I prefer camel case (with 1st letter capitalized) for class names, although I guess this doesn't hurt?

Reply:
Yea the spaces are not great either. To get dplyr to work we'll have to use the existing connection class names (e.g. `Microsoft Sql Server`) or submit a PR to dbplyr to add a new connection class name.